app/views/Settings.js

```diff
@@ -52,13 +54,42 @@ export const SettingsScreen = ({ navigation }) => {
     return () => {
       BackHandler.removeEventListener('hardwareBackPress', handleBackPress);
     };
-  }, [navigation]);
+  }, []);
```

Comment:
this revert could cause state issues.. please keep the previous [navigation]

Install the [vs code eslint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) so you can see warnings about this in your editor.